sources/packages/web/src/views/student/financial-aid-application/FullTimeApplication.vue

```diff
@@ -113,6 +114,15 @@ export default {
           OFFERINGS_DROPDOWN_KEY,
         );
       }
+      if (event.changed.component.key === OFFERINGS_DROPDOWN_KEY) {
```

Comment:
Right now we are loading a key/value pair to populate the Offerings dropdowns, did you considered or tested if we can load the key/value/OfferingDate instead and have access to the OfferingDate directly inside the form definition?

Reply:
as discussed, i first tried that, but its didn't worked